templates/barbican/config/00-default.conf

```diff
@@ -28,8 +28,8 @@ topic = barbican_notifications
 driver=messagingv2
 
 [oslo_policy]
-enforce_scope = true
-enforce_new_defaults = true
+enforce_scope = {{ .enableSecureRBAC }}
```

Comment:
Would we ever have a situation where we would want enforce_scope and enforce_new_defaults to be different from each other?

Reply:
I don't think so. We've only ever tested both off or both on.  I guess it's possible to do, but not something you'd want.

e.g. scope off and new defaults on would allow anyone with a reader role to access APIs intended for operators.  OTOH, scope on and new defaults off would enforce scope in the legacy policies (which are not written with scope in mind) so you'd probably get some weird behaviors.